src/Pages/MembershipApplication/membershipApplication.jsx

```diff
@@ -1,23 +1,24 @@
-import React, { useState } from 'react'
-import './register-page.css'
-import MembershipPlan from './MembershipPlan.js'
-import { memberApplicationState } from '../../Enums'
-import MembershipForm from './MembershipForm'
-import ConfirmationPage from './ConfirmationPage'
-import { CSSTransition } from 'react-transition-group'
+import React, { useState } from 'react';
```

Comment:
Why is this file changed?

Reply:
In the bump from react-scripts 3.4.3 to 4.0.0 there were some changes made to eslint. Now, JSX files have some extra linting done to them similar to how a js file is linted. We had to update these files to meet that standard so it could build properly without throwing any errors.

See the changelog for more information:
https://github.com/facebook/create-react-app/blob/master/CHANGELOG.md